beancount/parser/decimal.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BEANCOUNT_DECIMAL_H
```

Comment:
Remove this file; make the PyDecimal_IMPORT code in the initialization of the module (along with the comment), and place PyDec_FromCString in the lexer.l file./

Reply:
I mostly organize the code like this because of the global variable that holds a reference to the `Decimal` type. I was too lazy to have it defined `static` in one place and `external` in an header file. But I see the point. I will rework this.